### PR TITLE
Minor - Healthcare Desktop UI

### PR DESCRIPTION
**Pull request by system19**
Minor - Healthcare Desktop UI
Spelling error, Links fixed

### CODE REVIEW

**Review by nabinhait — CHANGES_REQUESTED** [review]
Any specific reason for changing the colour of the icons?

**Comment by system19** [comment]
No not really. I don't actually like colour. I just returned them all to the same colour they were before the last healthcare icon related PR. I like the idea of the healthcare Module being mostly one colour though.
The main part of this PR is that I found several Icons that didn't link. Clicking them would do nothing.

**Comment by raghukamath** [comment]
> No not really. I don't actually like colour. I just returned them all to the same colour they were before the last healthcare icon related PR. I like the idea of the healthcare Module being mostly one colour though.

Colour is subjective matter, some people like colours while others don't, however in this situation I guess we can keep different colour to make it a bit more consistent, because our desktop in other domains has various colours for icon background. adding color also makes it easy to spot icons and breaks the monotonous looks.

Can you please add a screenshot showing how it looks now. 

Thank you.

File: erpnext/config/desktop.py [hunk 1]
```diff
@@ -283,46 +283,53 @@ def get_data():
...
-			"color": "#2ecc71",
-			"icon": "fa fa-stethoscope",
-			"doctype": "Consultation",
-			"type": "link",
-			"label": _("Consultationt")
-        },
         {
 			"module_name": "Patient",
-			"color": "#6BE273",
+			"color": "#FF888B",
```
Comment (raghukamath) [inline erpnext/config/desktop.py]: Can you keep the colors as it is?

**Review by ghost — CHANGES_REQUESTED** [review]

File: erpnext/config/desktop.py [hunk 2]
```diff
@@ -283,46 +283,53 @@ def get_data():
...
+			"label": _("Patient Appointment"),
+			"hidden": 1
+        },
+        {
+			"module_name": "Consultation",
+			"color": "#FF888B",
+			"icon": "fa fa-stethoscope",
+			"doctype": "Consultation",
+			"type": "link",
+			"link": "List/Consultation",
```
Comment (raghukamath) [inline erpnext/config/desktop.py]: thanks for adding the links

**Comment by nabinhait** [comment]
@system19 As Raghu suggested, to keep consistency with other icons, I am merging this PR keeping the multiple colour.

**Comment by system19** [comment]
Sure, No problem :-)

**Pull request merged**